juniper_official/Solutions/RPM/rpm-probe-netconf.rule

```diff
@@ -0,0 +1,187 @@
+/*
+ * Detects RPM probe packet loss and RTT value using Netconf sensor.
+ * One inputs control detection
+ *
+ *   1) rpm-threshold-variable, is the threshold that causes the rule to report
+ *       an anomaly. By default it's 50000. This rule will set a dashboard
+ *       color to red when probe RTT value more than this threshold, otherwise
+ *       color is set to green.
+ */
+healthbot {
+    topic service.protocols {
+        description "Monitors RPM probe probe packet loss, RTT and notify anomalies";
+        synopsis "RPM probe analyzer";
+        rule rpm-probe-netconf {
+            keys [ owner test-name ];
+            synopsis "RPM probe packet loss monitor";
+            description "RPM probe";
+            /*
+             * Sensor configuration to get data from network devices.
+             */
+            sensor rpm-probe {
+                synopsis "RPM iAgent sensor definition";
+                description "RPM iAgent sensor to collect telemetry data from network device";
+                iAgent {
+                    file RPMprobe.yml;
+                    table RPMprobeTable;
+                    frequency 60s;
+                }
+            }
+            /*
+             * Fields defined using sensor path. Map the longer sensor names
+             * to the shorter field names used in the rules.
+             */
+            field loss-percentage {
+                sensor rpm-probe {
+                    path loss_percentage;
+                }
+                type integer;
+                description "RPM probe loss percentage";
+            }
+            field owner {
+                sensor rpm-probe {
+                    path owner;
+                }
+                type string;
+                description "RPM probe owner";
+            }
+            field probe-rt-delay {
+                sensor rpm-probe {
+                    path probe_rt_ave_delay;
+                }
+                type integer;
+                description "RPM probe rt delay";
+            }
+            field probe-rt-delay-dt {
+                formula {
+                    anomaly-detection {
+                        algorithm k-means;
+                        learning-period 7d;
+                        pattern-periodicity 1d,1h;
+                        field-name "$probe-rt-delay";
+                    }
+                }
+                type integer;
+            }
+            field test-name {
+                sensor rpm-probe {
+                    path test-name;
+                }
+                type string;
+                description "RPM probe test name";
+            }
+            /*
+             * Anomaly detection logic.
+             */
+            trigger rpm-probe-packet-loss-netconf {
```

Comment:
No need of ingest type in trigger name. Please remove -netconf.

Reply:
Modified the trigger name